vignette/Validation.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "Comparison"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Comparison}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>", 
+  fig.path = "man/figures/README-",
+  fig.width = 9, 
+  fig.height = 9
+)
+```
+
+```{r setup}
+library(hypatia)
+library(ggplot2)
+library(tidyverse)
+```
+
+```{r hypatia}
+
+# Create our population
+countryname = "Antigua and Barbuda"
+
+pop <- hypatia:::get_population(countryname)
```

Comment:
This suggests that get_population should be part of the API as you're advertising it to  users.